Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -281,7 +282,17 @@ public class CachedManifestLoader: ManifestLoading {
         guard let cachedManifestContent = String(data: cachedManifestData, encoding: .utf8) else {
             throw ManifestLoaderError.manifestCachingFailed(manifest, cachedManifestPath)
         }
+        do {
+            try await write(cachedManifestContent: cachedManifestContent, to: cachedManifestPath)
+        } catch let error as NIOFileSystem.FileSystemError {
+            guard error.code == .fileAlreadyExists else {
+                throw error
+            }
+            ServiceContext.current?.logger?.info("Encountered a `fileAlreadyExists` error, disregarding it")
```

Comment:
```suggestion
            if error.code == .fileAlreadyExists else {
                ServiceContext.current?.logger?.debug("The manifest at \(cachedManifestPath) is already cached, skipping...")
            } else {
              throw error
            }
            
```

I'd primarily change the logger to be `debug` and I would make the error message clearer. Personally, I feel `if-else` reads a bit better, but I don't feel strongly about it.